tests/api/api/test_feature_sets.py

```diff
@@ -57,6 +72,42 @@ def _assert_add_feature_set(client: TestClient, project, feature_set, versioned=
     return response.json()
 
 
+def _assert_put_feature_set(
```

Comment:
Name by the endpoint name not the http method, i.e. `_assert_store_feature_set` or `_assert_upsert_feature_set`